operator/src/main/resources/scripts/startServer.sh

```diff
@@ -126,11 +147,95 @@ function copySitCfg() {
   fi
 }
 
+# prepare mii server
+
+function prepareMIIServer() {
+
+  trace "Model-in-Image: Creating domain home."
+
+  # primordial domain contain the basic structures, security and other fmwconfig templated info
+  # domainzip only contains the domain configuration (config.xml jdbc/ jms/)
+  # Both are needed for the complete domain reconstruction
+
+  if [ ! -f /weblogic-operator/introspector/primordial_domainzip.secure ] ; then
+    trace "Domain Source Type is FromModel, the primordial model archive is missing, cannot start server"
+    exit 1
+  fi
+
+  if [ ! -f /weblogic-operator/introspector/domainzip.secure ] ; then
+    trace "Domain type is FromModel, the domain configuration archive is missing, cannot start server"
```

Comment:
"Domain type" -> "Domain source type"

Reply:
Needs to be "trace SEVERE" so it ends up in domain status

Reply:
@jshum2479 ?

Reply:
done

Reply:
@jshum2479, this doesn't look done

Reply:
It's in another PR 1515

Reply:
Oh, right 🤦‍♂